doc/ref/debug.xml

```diff
@@ -516,6 +516,18 @@ in microseconds.
   </Description>
   </ManSection>
 
+  <ManSection>
+  <Func Name="DisableMemoryLimits" Arg=''/>
+  <Returns>
+  nothing.
+  </Returns>
+  <Description>
+  Disables limits on the size of GAP's workspace. These are usually
+  set using the --warn-workspace and --maximal-workspace command line options.
```

Comment:
```suggestion
  set using the <C>--warn-workspace<C> and </C>--maximal-workspace</C> command line options.
```
Since we don't document these in the manual right now, perhaps it would be better to refer to them instead or additionally by their short names, which some users may be more familiar with anyhow?